packagehandlers/nugetpackagehandler.go

```diff
@@ -0,0 +1,35 @@
+package packagehandlers
+
+import (
+	"github.com/jfrog/frogbot/utils"
+	"github.com/jfrog/jfrog-client-go/utils/log"
+	"strings"
+)
+
+const dotnetPackageUpgradeExtraArg = "package"
+
+type NugetPackageHandler struct {
+	CommonPackageHandler
+}
+
+func (dotnet *NugetPackageHandler) UpdateDependency(vulnDetails *utils.VulnerabilityDetails) error {
```

Comment:
Usually in Go we use a shortcut as the method receiver name (nph). Yo can also use a meaningful name ("handler")